### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by firstpj**
-Created a feature branch.
 -Setup React environment using vite. 
-Created the to-do list…
-Added README.md file update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# React-to-do-list
\ No newline at end of file
+# React + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+
+Currently, two official plugins are available:
+
+- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
+- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
```
Comment (tajemouti) [inline README.md]: Kindly edit the README file to include the project details. A Professional README file is essential to explain what is uploaded and how we can install or use it.

**Review by tajemouti — CHANGES_REQUESTED** [review]
## Hi @firstpj

Good job so far!
There is an issue that you still need to work on but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Correct Gitflow ✅ 
- No linters errors✅ 
- The application works correctly ✅ 
- The application uses components ✅ 
- The application uses props and state ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @tajemouti in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by firstpj** [comment]
Thank you for the review. The require changes is noted.

**Review by tajemouti — APPROVED** [review]
# Approved :green_circle: 
Hi @firstpj, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] Implemented all the required changes :white_check_mark:   

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by firstpj** [comment]
Awesome, thank you so much!

**Comment by firstpj** [comment]
https://microverse.zoom.us/j/95543253837?pwd=eXFUdUVJWFRRQWFsTkRVK1FuSWpLQT09

**Comment by firstpj** [comment]
https://microverse.zoom.us/j/95543253837?pwd=eXFUdUVJWFRRQWFsTkRVK1FuSWpLQT09

**Comment by firstpj** [comment]
https://microverse.zoom.us/j/97530658650?pwd=UTlnSGlwbGhVaitMTnh2MXZDNmIzQT09

**Pull request merged**